### ISSUE TITLE
feat(select): disable  Staggered animation on Select tag

### ISSUE THREAD

**Issue by BjornBakke** [fremtind/jokul#713]
Staggered animation on the Select dropdown should be optional. (Motion Sensitivity)

**Comment by joms** [comment]
@BjornBakke Would that mean being able to disable the animation of options inside the `select` through props? Or should one be able to disable animation on the `select` altogether?

**Comment by Steinop** [comment]
The staggered animation should not be there at all. The animation does not look good. New and less annoying animation will be added to the new select / dropdown menu.
